### ISSUE TITLE
Wrong index in 2NN/L1/23

### ISSUE THREAD

**Issue by bartmate** [udacity/dlnd-issue-reports#380]
During the calculation of the partial derivate of E respect to w_j, the index turns from "j" to "i" at the end. It should remain "j".

**Comment by Ortalarel** [comment]
Can you please direct us to the specific lesson you are referring to?

**Comment by bartmate** [comment]
2NN/L1/23:
2.Neural Networks / Lesson 1 - Introduction to Neural Networks / 23. Gradient Descent

However I cannot see the issue any more. Is it possible that it was fixed in the last 3 weeks?

**Comment by bartmate** [comment]
See my new comment.

**Comment by Ortalarel** [comment]
yes, this was fixed. Thank you
